### DISCUSSION TITLE
Give support for running application

### DISCUSSION THREAD

**Discussion by danies8** [quartznet/quartznet#2257] [Q&A]
Hi,
Is there any body who could give support to a given application ?
The application frozen from time to time when using quartznet.

**Comment by da3dsoul** [comment]
https://stackoverflow.com/help/how-to-ask

**Comment by danies8** [comment]
@da3dsoul 
I'm looking for someone who could help us to find
Why our scheduler is frozen from time to time?
(With money)
Can you help?
